### DISCUSSION TITLE
1.0.0.alpha

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#46] [Announcements]
# Pre-release 1.0.0.alpha

Pre-release for public test. Currently implements only 16S Illumina sequencing of Birds and Mammals prepared according to the [Dobrovolny method](https://pubmed.ncbi.nlm.nih.gov/30309555/).

Check the [documentation]([bio-raum.github.io/FooDMe2/](https://bio-raum.github.io/FooDMe2/)) for installation and usage.

This release is for test purposes, things will probably break and the documentation is lacking in many parts.
Should you encounter any problem or have questions, please raise an issue directly on the repository or contact the developpers directly.

<hr /><em>This discussion was created from the release <a href='https://github.com/bio-raum/FooDMe2/releases/tag/1.0.0.alpha'>1.0.0.alpha</a>.</em>

**Comment by gregdenay** [comment]
Already some (minor) problems :dizzy_face:

- The `main` branch is not automatically fetched by the workflow, need to specify it with `-r main`. Easy to fix though, I added `defaultBranch = 'main'` to the `manifest` block of `nextflow.config`. In the 1.0.0.alpha omitting this will result in an error message asking the user to specify `-r`.

- The nextflow 'header' on piplein eexecution is still mentionning version 0.2 (the commit points to 1.0.0.alpha). Where does nextflow gets this info from? Also there are some equal signs missing this not important but annoying me everytime.
```
    ===============================================================================
    FooDMe2 Pipeline | Profiling eukaryotic taxa from mitochondrial amplicons | version 0.2
    ===============================================================================
```
@marchoeppner

**Reply by gregdenay** [reply]
Edit the default brahcn of the **repo** was dev, I modified it to main. 
Having to specify `-r` is apparently a feature so for the record:  `-r 1.0.0.alpha` to run the test release

**Reply by marchoeppner** [reply]
Yes, earlier versions of Nextflow would automatically use the latest commit to whatever the default branch was, but I guess they changed that for whatever reason.

**Reply by marchoeppner** [reply]
and I changed the version printed to the screen  (dev branch) - this comes from here:

https://github.com/bio-raum/FooDMe2/blob/b818ec5a279733c24b34fd2c2d28eddb6b029d49/nextflow.config#L85
